src/main/java/com/toptrumps/cli/CommandLineView.java

```diff
@@ -24,8 +24,9 @@
 
     public void showWelcomeMessage() {
         printWelcomeBanner();
-        typePrint(SLOW_SPEED, "\nTo start a new game, enter f \nTo see game statistics, enter s \nTo quit, enter q", false);
-        Logger.getInstance().logToFileIfEnabled("Game started");
+
+        typePrint(SLOW_SPEED, "\nTo start a new game, enter f \nTo see game statistics, enter s \nTo quit, enter q");
```

Comment:
I missed this one - please add false as the final argument to typePrint

Reply:
Done. I hate merges